tests/utils_tests/test_http.py

```diff
@@ -139,6 +139,10 @@ def test_is_safe_url(self):
         self.assertTrue(http.is_safe_url('/confirm/me@example.com'))
         # Basic auth without host is not allowed.
         self.assertFalse(http.is_safe_url(r'http://testserver\@example.com'))
+        # HTTPS only allowed when secure=True.
+        self.assertFalse(http.is_safe_url('http://example.com', 'example.com', secure=True))
+        # HTTP allowed by default.
+        self.assertTrue(http.is_safe_url('http://example.com', 'example.com'))
```

Comment:
could you please more test cases to check if url secured or not, please see my example of test case here https://github.com/django/django/pull/6928/files#diff-01a1ff7510f29ab8ea151390573f5736R143